tools/generate_tables_sql.php

```diff
@@ -68,26 +68,31 @@
 
             //generate specific column definitions for specific types of HTML elements
             default:
-                if($bits[1] == "") {
+                if((array_key_exists(1,$bits) ? $bits[1] : "") == "") {
                     continue;
```

Comment:
Same suggestion:
`if (empty($bits[1]))`
Or if we want to keep `$bits[1] == ""` then 
`if ( isset($bits[1] ) && $bits[1] == "")`